README.md

```diff
@@ -5,28 +5,102 @@ This guide is provided as is, it may not work with everyone's Linux setup and ma
+   Main PID: 79790 (start.sh)
+      Tasks: 12 (limit: 14125)
+     Memory: 42.3M
+        CPU: 1.612s
+     CGroup: /user.slice/user-1000.slice/user@1000.service/app.slice/peacock.service
+             ├─79790 /bin/bash /home/deck/build/linux-steam-setup/start.sh
+             └─79804 ./node/bin/node ./Peacock/chunk0.js
+
+Dec 20 16:40:07 steamdeck start.sh[79804]:  ░██████████  ░██████    ░███████████ ░███         ░███      ░███░███          ░███████
+Dec 20 16:40:07 steamdeck start.sh[79804]:  ░███░░░░░░   ░███░░█    ░███░░░░░███ ░███         ░███      ░███░███          ░███░░███
+Dec 20 16:40:07 steamdeck start.sh[79804]:  ░███         ░███ ░   █ ░███    ░███ ░░███     ███░░███     ███ ░░███     ███ ░███ ░░███
+Dec 20 16:40:07 steamdeck start.sh[79804]:  █████        ██████████ █████   █████ ░░█████████  ░░░███████░   ░░█████████  █████ ░░████
+Dec 20 16:40:07 steamdeck start.sh[79804]: ░░░░░        ░░░░░░░░░░ ░░░░░   ░░░░░   ░░░░░░░░░     ░░░░░░░      ░░░░░░░░░  ░░░░░   ░░░░
+Dec 20 16:40:07 steamdeck start.sh[79804]: [16:40:07:100] [Info] This is Peacock v6.5.6 (rev 6560), with Node v18.15.0.
+Dec 20 16:40:07 steamdeck start.sh[79804]: [16:40:07:102] [Info] Booting Peacock internal services - this may take a moment.
+Dec 20 16:40:07 steamdeck start.sh[79804]: [16:40:07:104] [Info] Server started.
+Dec 20 16:40:07 steamdeck start.sh[79804]: [16:40:07:271] [Debug] Discovered 161 escalation groups.
+Dec 20 16:40:07 steamdeck start.sh[79804]: [16:40:07:591] [Debug] Peacock is up to date.
+```
+
+The most important bit there is "Server started" and "Peacock is up to date". If Peacock is not up to date, remove the Peacock folder and restart the service with `systemctl --user restart peacock`
+
+## Adjust your launch options in Steam
+
+Ensure this is copied exactly, you can use the clipboard icon next to this snippet to copy it (assuming you're browsing this on github.com)
+
+```bash
+bash -c 'exec "${@/Launcher.exe/WineLaunch.bat}"' -- %command%
+```
+
+Then right-click on the game in Steam -> Properties and paste it in the "Launch Options" box. Steam Deck users should use the cog icon on the game detail view then continue as needed; but due to the awkward keyboard you might find it easier to do in desktop mode.
+
+## Start Hitman 3
+Run from anywhere, the game should start alongside the Peacock Patcher. If your game starts up full-screen, then you will have to alt-tab out.
+
+Steam Deck users can access the patcher by pressing the Steam button and navigating to the active windows at the bottom of the "HITMAN 3" section.
+
+## Edit Peacock Patcher Server address to `127.0.0.1:3000`
+
+It's not clear that you can directly edit the combo box that says "Peacock Local" but you can, alternatively add an `/etc/hosts` entry for `127.0.0.1    localhost`. If you're likely to switch back and forth between IOI servers and Peacock then you'd be better off setting an entry in `/etc/hosts`.
```

Comment:
This part must make it clear that you *need* to edit the value in the patcher, as by default Peacock and the patcher are configured to run on port 80, or else you won't be able to get past title screen.
Because Linux restricts access ports under 1024 to non-root user, we decided for the sake of simplicity to use port 3000 and so requires the user to change the value on the patcher once to `127.0.0.1:3000`